test/Libraries/Microsoft.Extensions.AI.Evaluation.Reporting.Tests/EmbeddingTests.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using Microsoft.Extensions.AI.Evaluation.Reporting.Formats.Html;
+using Microsoft.TestUtilities;
+using Xunit;
+
+namespace Microsoft.Extensions.AI.Evaluation.Reporting.Tests;
+
+public class EmbeddingTests
+{
+    [ConditionalFact]
+    public void CIBuildsMustIncludeEmbeddedHTML()
+    {
+        // TF_BUILD should be set in our CI pipeline
+        if (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("TF_BUILD")))
```

Comment:
Should we say `if (!Settings.Current.Configured)` instead? I'd assume that we also want the test to fail locally if this happens to be missing?

Reply:
I thought about this. I couldn't think of a way to have it work locally and also stay out of the way when people don't have EnableEvaluationReportBuild enabled.

Reply:
Could we perhaps also require them to pass when `Settings.Configured` is true?

Reply:
I'm happy to iterate on this, but let's do that in another PR